docs/pages/guides/replicating-onchain-state.mdx

```diff
@@ -1,7 +1,17 @@
+import { Callout } from "nextra/components";
+
 # Replicating onchain state
 
 This guide walks through how you might recreate onchain state from MUD's Store events. We use this pattern in our sync stack to hydrate a client or indexer from blockchain logs fetched from an RPC.
 
+<Callout type="info" emoji="ⓘ">
+  In most cases, you would use a pre-written [template](/templates), which includes code to synchronize information.
+  If that is not an option, you might be able to use [an indexer](/indexer) to retrieve the onchain data.
+
+This guide is mostly provided as an illustration, and to simplify the writing of future clients.
+
+</Callout>
+
```

Comment:
Do we really need a callout?

```suggestion
# Replicating onchain state

This guide walks through how you might recreate onchain state from MUD's Store events. We use this pattern in our sync stack to hydrate a client or indexer from blockchain logs fetched from an RPC.

If you are using a MUD template, one of our sync packages, or indexers, all of this is already done for you. This guide is meant to demonstrate how the Store protocol works and show how you might implement a sync strategy or indexer of your own.

```

Reply:
@ludns , I don't feel strongly either way - can you and @holic discuss?